service.py

```diff
@@ -49,7 +50,10 @@ def do_GET(self):
             'quiet': True,
             'simulate': True
         }
+
         if url.path == '/process':
+            if proxy_url and proxy_url[0] != '':
```

Comment:
Isn't `''` falsy anyway?

Reply:
Had to fiddle with this a bit; first one checks for presence for `proxy_url` (can be `list` or `None`), 2nd one checks the actual value of the param as we know it's present now

Reply:
How come we can even get an array like `['']` though? That seems like a bug

Reply:
Sanity check I suppose, make sure that the `proxy_url` query param was provided, and that it's not an empty string for whatever reason. 

My other changes (vivi-service-ytdl) don't send an empty string as the `proxy_url` value if that's what seems buggy, the implementation higher up has its own checks